src/app/models/Tags.ts

```diff
@@ -0,0 +1,4 @@
+export interface Tags {
+    id: number;
+    tagName: string;
```

Comment:
too big indent, get prettier!

Reply:
used the alt shift F command to correct.